### DISCUSSION TITLE
DateInput within a Modal - missing withinPortal?

### DISCUSSION THREAD

**Discussion by chvid** [mantinedev/mantine#4696] [Q&A] [answered]
HI.

Great framework!

I am trying to display a date input (that popups a date picker) within a modal dialog. However the date picker is completely obscured trying to fit in the small dialog.

Unlike Select controls DateInput doesn't seem to have a withinPortal property. What the right way to do this?

```typescript
import { useDisclosure } from "@mantine/hooks";
import { Button, MantineProvider, Modal } from "@mantine/core";
import { DatePickerInput, DateValue } from "@mantine/dates";
import { useState } from "react";

function App() {
  const [opened, { toggle, close }] = useDisclosure(false);
  const [value, setValue] = useState<DateValue>(new Date());

  return (
    <MantineProvider>
      <Button onClick={toggle}>Click me!</Button>
      <Modal opened={opened} onClose={close} title="Date Input Test">
        <DatePickerInput value={value} onChange={setValue} label="Date input" placeholder="Date input"/>
      </Modal>
    </MantineProvider>
  );
}

export default App;
``` 

I am using the following versions:

 "@mantine/core": "^6.0.19",
 "@mantine/dates": "^6.0.19",
 "@mantine/hooks": "^6.0.19",

**Comment by rtivital** [comment, marked as answer]
`< DatePickerInput popoverProps={{ withinPortal: true }} />`

**Comment by chvid** [comment]
Thanks a bundle - it works.
